src/collectors/diskspace/diskspace.py

```diff
@@ -156,8 +161,12 @@ def get_file_systems(self):
                                        mount_point)
                         continue
 
-                    if (major, minor) in result:
+                    if (major, minor) in result \
+                            and mount_point not in self.no_major_minor:
```

Comment:
The default for `self.no_major_minor` seems to be `'nfs'`, which is more like a type rather than a mount point itself. Am I misreading something here, or did you intend to use `fs_type` here instead?